src/screens/SignIn/validation-schema/validation-schema.ts

```diff
@@ -0,0 +1,14 @@
+import { object, string } from 'yup';
+
+const SignInSchema = object().shape({
+  email: string()
+    .trim()
+    .required('Email is required')
+    .email('Please enter a valid email'),
+  password: string()
+    .trim()
+    .required('Password is required')
+    .min(6, `Password must be 6 characters or longer`),
```

Comment:
Maybe it will be better if we create SignInValidationRule & SignInValidationMessage enums similar to our Web application?

Reply:
I did it at the beginning, but then we decided that this structure is too complicated for our project.